redirect/routes.php

```diff
@@ -71,10 +71,12 @@ class Route {
         'anal'         => [
             'description' => 'Analysis für Informatiker',
             'target'      => 'https://www-m5.ma.tum.de/Allgemeines/MA0902_2017W',
+            'moodle_id'   => '36704',
```

Comment:
To camelCase or not is the question - right?

Reply:
I onow what I prefer :blush: But we should probably stick to the language / framework conventions - whatever this might be in case of PHP

Reply:
What do you prefer?

Reply:
Snake case :snake: